apidef/oas/event.go

```diff
@@ -116,10 +140,28 @@ func (e *EventHandler) GetWebhookConf() apidef.WebHookHandlerConf {
 	}
 }
 
+// JSVMEvent represents a JavaScript VM event configuration for event handlers.
+type JSVMEvent struct {
```

Comment:
Can you sanitize the order a bit please, type definitions should be in front/before functions, close to the top of the file next to other (grouping). Move this block there

Reply:
I would normally agree, but in this case the context is relevant, so it makes sense to break it up. It is also consistent to exisiting implementation. It's sometimes helpful from developer perspective to keep the context grouped instead of the kind of the code